frontend/src/app/main/ui/workspace.cljs

```diff
@@ -70,6 +71,7 @@
 
        [:& viewport {:file file
                      :local local
+                     :wstate wstate
```

Comment:
It has the old name `wstate` I think better to call it `global` to be more coherent with its name in the state

Reply:
We will do it in a different MR